### PR TITLE
Replace all vtctlclient usage for vreplication commands

### LINKED ISSUE #14206: Update any vtctlclient VReplication command usage in the Vitess website docs for  v18+

### PR DESCRIPTION
**Pull request by mattlord**
Fixes: https://github.com/vitessio/vitess/issues/14206

>**Note**
> Once I get approval on the changes, then I'll copy them to the v18 docs.

### CODE REVIEW

File: content/en/docs/19.0/reference/features/connection-pools.md [hunk 1]
```diff
@@ -89,7 +89,6 @@ As a result, connection pools should be sized mindful of the capacity of the und
 #### `--db_allprivs_user` 
 
  * (default 'vt_allprivs')
- * Created on demand by `vtctlclient ExecuteFetchAsAllPrivs`
```
Comment (frouioui) [inline content/en/docs/19.0/reference/features/connection-pools.md:92]: Why are we not keeping this for `vtctldclient`,

File: content/en/docs/19.0/user-guides/migration/materialize.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ All of the command options and parameters are listed in our [reference page for
 Let's start by loading some sample data:
 
 ```bash
-$ mysql < ../common/insert_commerce_data.sql
```
Comment (frouioui) [inline content/en/docs/19.0/user-guides/migration/materialize.md:34]: Should we remove the `$` in all shell snippet?

File: content/en/docs/19.0/reference/features/connection-pools.md (hunk 1 above)
Comment (mattlord) [reply content/en/docs/19.0/reference/features/connection-pools.md:92]: There's no `ExecuteFetchAsAllPrivs` vtctldclient command. I'm not sure that we need one either.

/cc @ajm188

File: content/en/docs/19.0/user-guides/migration/materialize.md (hunk 2 above)
Comment (mattlord) [reply content/en/docs/19.0/user-guides/migration/materialize.md:34]: Not necessary, but the general standard (markdown linter e.g.) is that you do NOT include the prompt if the command is not run and thus the output not shown (but instead you're only showing the command that the reader should run). That's why I removed it here, as we do not include the command output.

**Comment by shlomi-noach** [comment]
`OnlineDDL` related work here: https://github.com/vitessio/website/compare/docs-onlineddl-vtctldclient?expand=1

**Review by deepthi — APPROVED** [review]

**Review by rohit-nayak-ps — APPROVED** [review]
Nice, thanks for the ton of work required to do this!

File: content/en/docs/19.0/reference/features/connection-pools.md (hunk 1 above)
Comment (ajm188) [reply content/en/docs/19.0/reference/features/connection-pools.md:92]: there's also no equivalent RPC for vtctld server. this was deliberate, as me, @deepthi and others agreed it was no longer needed

**Pull request merged**